src/v2/computations-and-formatting.ts

```diff
@@ -374,8 +374,8 @@ export function formatReserves(
         reserve.totalPrincipalStableDebt,
         calculateCompoundedInterest(
           reserve.averageStableRate,
-          reserve.stableDebtLastUpdateTimestamp,
-          currentTimestamp
+          currentTimestamp,
```

Comment:
reverted this pr previously as i thought it broke sth, but in fact is was broken before - the pow implementation was just less fragile to this kind of error (parameters were switched so the exponent was negative).